docs/packaging/package-contract.md

```
@@ -0,0 +1,364 @@
+from vibe.errors import ...      # the public exception hierarchy (§6)
+```
+
+`vibe` (top-level) is **stable across downstream repos** for v0. Anything reachable
+without a leading underscore from `vibe`, `vibe.config`, `vibe.errors`, and the
+documented integration entrypoints is part of the promise (§7).
+
+### 1.2 Integration submodule convention
+
+Integrations live under a single, predictable namespace:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)